packages/codegen/src/inheritance.ts

```diff
@@ -11,7 +11,7 @@ import { fail } from "./utils";
 export function applyInheritanceUpdates(config: Config, db: DbMetadata): void {
   const { entities, entitiesByName } = db;
   setClassTableInheritance(entities, entitiesByName);
-  expandSingleTableInheritance(config, entities);
+  expandSingleTableInheritance(config, entitiesByName, entities);
```

Comment:
Unrelated fix up, the "expand STI" logic wasn't adding the `TaskOld` / `TaskNew` inferred/discovered metas to `entitiesByName`, so we pass that along to get updated.